src/jit/compiler.h

```diff
@@ -517,7 +517,7 @@ class LclVarDsc
     {
         if (slotNum == 0)
         {
-            return lvArgReg;
+            return _lvArgReg;
```

Comment:
`GetArgReg()` is doing `(regNumber)` cast, so it should be 
```
return GetArgReg(); 
```
as it was (and what is done in the next condition for `slotNum == 1`
or
```
return (regNumber)_lvArgReg; 
```

as I said before it doesn't really matter in this case.